h/security/policy/top_level.py

```diff
@@ -13,75 +10,34 @@
 from h.security.policy.helpers import is_api_request
 
 
-@implementer(ISecurityPolicy)
```

Comment:
This `@implementer(ISecurityPolicy)` was added to some of our security policies but not others. Security policies don't need to have this and the examples in the Pyramid docs don't have it. So I'm just removing it from all of them.